src/homework5/chessBoard.java

```diff
@@ -0,0 +1,16 @@
+package homework5;
+
+import java.util.Arrays;
+
+public class chessBoard {
+    public static void main(String[] args) {
+        String [][] chessBoard = new String[8][8];
+        for (int i = 0; i< chessBoard.length; i++) {
+            for (int j = 0; j < chessBoard[0].length; j++) {
+                if ((i + j) % 2 == 0) chessBoard[i][j] = "W";
+                else chessBoard[i][j] = "B";
+            }
+        }
+        System.out.println(Arrays.deepToString(chessBoard)+ " ");
```

Comment:
`for (String[] lines : chessBoard) {
            System.out.println(Arrays.deepToString(lines));
        }`